tap_bing_ads/__init__.py

```diff
@@ -836,14 +830,12 @@ def get_report_request_id(client, account_id, report_stream, report_name,
     return client.SubmitGenerateReport(report_request)
 
 
-def build_report_request(client, account_id, report_stream, report_name,
+def build_report_request(client, account_id, report_stream, report_name,    # pylint: disable=too-many-arguments
                          start_date, end_date):
-    LOGGER.info(
-        'Syncing report for account {}: {} - from {} to {}'
-        .format(account_id, report_name, start_date, end_date)
-    )
+    LOGGER.info('Syncing report for account %s: %s - from %s to %s',
+                account_id, report_name, str(start_date), str(end_date))
 
-    report_request = client.factory.create('{}Request'.format(report_name))
+    report_request = client.factory.create(f'{report_name}Request'.format(report_name)) # pylint: disable=consider-using-f-string
```

Comment:
.format is not required if you are using f-stream. Please check the usage of f-stream